core/client/src/main/java/alluxio/client/block/LocalBlockInStream.java

```diff
@@ -118,6 +120,21 @@ public int directRead(byte[] b, int off, int len) throws IOException {
 
   @Override
   protected void incrementBytesReadMetric(int bytes) {
-    ClientContext.getClientMetrics().incBytesReadLocal(bytes);
+    Metrics.BYTES_READ_LOCAL.inc(bytes);
+  }
+
+  /**
+   * Class that contains metrics about LocalBlockInStream.
+   */
+  @ThreadSafe
+  private static final class Metrics {
```

Comment:
I feel it makes it harder to add metrics if we group all metrics class together like what we did earlier.  I actually prefer to inlining the counter and remove this Metrics class if we have an easy way to access all the metrics for an application. One way is to add a web handler display *all* metrics without much overhead. We can still have out current webUI which displays important ones.